lib/transport_web/controllers/user_controller.ex

```diff
@@ -21,6 +23,41 @@ defmodule TransportWeb.UserController do
     end
   end
 
+  def organization_form(conn, _params) do
```

Comment:
Wouldn't it be clearer a ```TransportWeb.User.OrganizationController``` ? That way controller action name will simply be ```new```, which feels more natural than ```form```.

Reply:
Yep, I'll open an issue for that.
This PR is already too big.

Reply:
OK